src/public/resolvers/queries/Collection.ts

```diff
@@ -52,3 +52,17 @@ export async function getCollections(
     collections,
   };
 }
+
+export async function resolveReference({ slug }, { dataLoaders }) {
+  if (!slug) {
+    throw new UserInputError('slug required');
+  }
```

Comment:
Should the error message be a bit more descriptive here? If a slug is missing, do graph clients need to know they couldn't resolve a collection entity as opposed to, for example, a syndicated article? Or a slug is a slug and it doesn't really matter?